.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: python
-sudo: false
+
+dist: xenial
+
 python:
   - 2.7
-  - 3.4
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8-dev
   - pypy-5.4.1
```

Comment:
Make that pypy2.7-6.0.0 and it should work with dist: xenial.

(You can also add pypy3.5-6.0.0 if you want to.)

Reply:
Thanks a lot!